tests/framework/contracts/operations.go

```diff
@@ -88,5 +88,6 @@ type RestAPIOperator interface {
 }
 
 type TestOperator interface {
```

Comment:
what if you just called it the `Setup` interface?

Reply:
yes, that sounds better.